### ISSUE TITLE
Hero Templating

### ISSUE THREAD

**Issue by msowka-ninja** [kataras/go-template#11]
As Iris should be the fastest web-framework I propose to add Hero-Templating
https://github.com/shiyanhui/hero

A benchmark which proves hero to be the fastest golang templating engine:
https://github.com/SlinSo/goTemplateBenchmark

**Comment by kataras** [comment]
Hello @msowka-ninja , thanks for your post and request.

Hero is pre-compiled template engine, a pre-compiled template engine is a command line tool at the end., so it couldn't be adapted here because it will break the existence flow I've selected for this repository. This repository contains a common interface of famous fully featured template parsers. 

I saw the benchmarks and I'm thinking to adapt the [jet](https://github.com/CloudyKit/jet/wiki/Jet-template-syntax) template parser instead, it's very good template parser and maybe the best(as far as I have seen) for the Go Programming Language.

**Comment by mayowa** [comment]
> I saw the benchmarks and I'm thinking to adapt the jet template parser instead, it's very good template parser and maybe the best(as far as I have seen) for the Go Programming Language.

I agree, any plans to add it soon? I could do a pull request if that would help

**Comment by kataras** [comment]
I'm working on different things lately, but I believe I could make it happens until the next week. If you can do it sooner, please push a PR !

**Comment by dxfeng10** [comment]
@kataras any update on Jet implementation ?
